Objects/unicodeobject.c

```diff
@@ -15268,7 +15487,7 @@ PyTypeObject PyUnicode_Type = {
     (reprfunc) unicode_str,       /* tp_str */
     PyObject_GenericGetAttr,      /* tp_getattro */
     0,                            /* tp_setattro */
-    0,                            /* tp_as_buffer */
+    &unicode_as_buffer,           /* tp_as_buffer */
```

Comment:
Could this addition be the issue with the tests?

Reply:
Ah yes, there is an additional method on `__str__` namely `__release_buffer__` being created because of this addition.

Reply:
> We have the following assertion error: UserString should have all the methods of str

What did you do to get this error?

Reply:
Ah, test_collections was failing, I just saw it on the CI. I pushed a fix for that: I implemented `collections.UserStrings.__release_buffer__()` as `raise NotImplementedError`.

Reply:
Ah sorry I wasn't clear! it was about the CI test failures.

Reply:
I saw the fix. Thanks!